.github/ci/scripts/symfony-require-dev.sh

```diff
@@ -0,0 +1,4 @@
+SYMFONY_VERSION=${SYMFONY_VERSION:-local}
```

Comment:
What's `local` for a version?

Reply:
@jdreesen you are right, but actually it doesn't hurt as there is still the empty check for `SYMFONY_VERSION` in the workflow.yml's. 
@dvesh3 I think we should fix that anyway, not that it causes confusion in future 😊

Reply:
yes, a leftover from debugging. fixed it on 10.x branch 😉 thanks!